content/docs/get-started/index.md

```diff
@@ -4,97 +4,214 @@ description: 'Learn how you can use MLEM to easily manage and deploy models'
 
 # Get Started
 
-Assuming MLEM is already [installed](/doc/install) in your active python
-environment, let's initialize it by running `mlem init` inside a Git project:
+We assume MLEM is already [installed](/doc/install) in your active Python
+environment, as well as `pandas` and `sklearn`. If not, you can follow the
+instructions below:
 
 <details>
 
 ### ⚙️ Expand for setup instructions
 
-If you want to follow along with this tutorial and try MLEM, you can use our
-[example repo](https://github.com/iterative/example-mlem-get-started). You'll
-need to [fork] it first (so you can push models). Then clone it locally:
-
-[fork]: https://docs.github.com/en/get-started/quickstart/fork-a-repo
-
-```cli
-$ git clone <your fork>
-$ cd example-mlem-get-started
-```
-
-Next let's create an isolated virtual environment to cleanly install all the
-requirements (including MLEM) there:
+Let's create a separate folder and an isolated virtual environment to cleanly
+install all the requirements (including MLEM) there:
 
 ```cli
+$ mkdir mlem-get-started
+$ cd mlem-get-started
 $ python3 -m venv .venv
 $ source .venv/bin/activate
-$ pip install -r requirements.txt
+$ pip install pandas scikit-learn mlem[fastapi,heroku]
 ```
 
 </details>
 
-```cli
-$ mlem init
-```
+## Saving your ML model
 
-A few [internal files](/doc/user-guide/project-structure) will be created:
+To enable all kinds of productionization scenarios MLEM supports, we first need
+to save a ML model with MLEM.
 
-```cli
-$ tree .mlem
-.mlem
-└─── config.yaml
-```
+Let's take a look at the following Python script:
 
-Now you’re ready to MLEM!
+```py
+# train.py
+from sklearn.datasets import load_iris
+from sklearn.ensemble import RandomForestClassifier
 
-In our
-[example repository](https://github.com/iterative/example-mlem-get-started),
-you'll find tags for each step we take in the different sections of this
-tutorial. You can just see what is going on there or reproduce everything
-yourself and compare. In the different `Get Started` sections, those tags will
-be marked with ⛳ emoji. Click on it to expand the section and see the `git`
-commands to run if you are following along. Just like this Git tag that
-concludes this section:
+from mlem.api import save
 
-<details>
 
-# ⛳ MLEM init
+data, y = load_iris(return_X_y=True, as_frame=True)
+rf = RandomForestClassifier(
+    n_jobs=2,
+    random_state=42,
+)
+rf.fit(data, y)
+save(
+    rf,
+    "models/rf",
+    sample_data=data,
+)
+```
+
+Here we load well-known iris dataset with sklearn and train a simple classifier.
+But instead of pickling the model we saved it with MLEM (check out the
+[full list of ML frameworks](/doc/object-reference/model) that MLEM supports.)
```

Comment:
```suggestion
Here, we load a well-known
[Iris flower dataset](https://archive.ics.uci.edu/ml/datasets/iris) with scikit-learn
and train a simple classifier. But instead of pickling the model, we save it with
MLEM (check out the full list of supported
[ML frameworks](/doc/object-reference/model)).
```